api-catalog-ui/frontend/src/components/App/App.jsx

```diff
@@ -35,9 +40,17 @@ class App extends Component {
                     <ErrorContainer />
                     <Suspense fallback={<Spinner isLoading={isLoading} />}>
                         <Router history={history}>
+                            {/* eslint-disable-next-line react/jsx-no-useless-fragment */}
```

Comment:
I don't see what's different in the next line from before, was this being flagged by the eslint?

Reply:
yes looks like it is flagged by aslant, I tried to remove it and it complains. Not sure how why itwas working before